skill-tree/pom.xml

```diff
@@ -207,6 +207,41 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+
+                <configuration>
+                    <skip>${skip-ut}</skip>
+                    <skip>${skip-tests}</skip>
+                    <includes>
+                        <include>**/unit/*.java</include>
+                    </includes>
+
+                </configuration>
+
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <skip>${skip-tests}</skip>
+                    <includes>
+                        <include>**/integration/*.java</include>
```

Comment:
please state the reason why we have added these new packages ?

Reply:
These are to run integration and unit tests in separate stages... 
failsafe is generally used to run integration tests and surefire is used to run unit tests - 
https://maven.apache.org/surefire/maven-failsafe-plugin/faq.html

Reply:
Resolved this offline

Reply:
Discussed, and okay with this